src/static/js/chapter.js

```diff
@@ -199,8 +208,10 @@ function setDiscussionCount() {
           if (isNaN(comments)) {
             return;
           }
+          var commentSingular = getElementContent('#comment-singular', 'comment');
+          var commentPlural = getElementContent('#comment-plural', 'comments');
           var el = document.getElementById('num_comments');
-          el.innerText = comments + ' ' + (comments == 1 ? 'comment' : 'comments');
+          el.innerText = comments + ' ' + (comments == 1 ? commentSingular : commentPlural);

```

Comment:
Actually would a better way be to have the comment/comments text in the appropriate place in the template and then show the appropriate one with JavaScript-initiated CSS?

Seems better than injecting content from one element to another via JavaScript.

But that would definitely require changing the English template as not backwards compatible like your one is. But I'm fine with that.